aws_lambda_builders/workflows/nodejs_npm_esbuild/actions.py

```diff
@@ -103,65 +109,62 @@ def _should_bundle_deps_externally(self) -> bool:
         return self._skip_deps or "./node_modules/*" in self._bundler_config.get(EXTERNAL_KEY, [])
 
 
-class EsbuildCheckVersionAction(BaseAction):
-    """
-    A Lambda Builder Action that verifies that esbuild is a version supported by sam accelerate
+def check_minimum_esbuild_version(
```

Comment:
the logic of the version checking did not change

Reply:
I don't understand the benefit of converting this from an `Action` to its own method. Lambda Builders written in this way where each step in build process is defined by actions, and actions are defined during workflow creation step. Once they are created, they can be executed in order. I feel like this is somewhat against the general design of this library.

Reply:
if two actions are such that they **always** need to be appended together (and in a certain order), they probably don't need to be separate actions. There is no case when the bundle action with the `--external` flag should be used without first having to check that their esbuild version supports it. In my opinion it is safer (we can't miss it), and cleaner, to have this check happen on the bundle action rather than as a separate action. 
I hope this doesn't go against the design of the library. I view bundling as **the** action, while the check is just something that needs to happen as part of the bundling.

Reply:
> if two actions are such that they always need to be appended together (and in a certain order)

I agree but then they should be combined as single action, if we won't need them individually.

Regarding the rest of your comment; this library can also be designed in a way that each action can be its own method, and workflows can call those methods depending on the parameters it has (like if it needs to download dependencies, or need to combine them etc.). But it will be hard to test the behavior with certain workflows, since we may need to manually check if a function is called or not. In current way, we can easily confirm behaviors in the tests by just looking at the order of the actions.  And confirming the behavior of the actions is easy, because they are isolated into their own generic classes.

This one change might be OK, but I want us to be careful about making this as an habit and move away from the 'Actions'.

Reply:
@mndeveci and I discussed this offline and it seems like there was a misunderstanding, and we're good with this change. The two actions were indeed combined into a single action.